test/acceptance/features/steps/steps.py

```diff
@@ -257,6 +258,14 @@ def sbo_is_ready(context, sbr_name):
     sbo_jq_is(context, '.status.conditions[] | select(.type=="CollectionReady").status', sbr_name, 'True')
     sbo_jq_is(context, '.status.conditions[] | select(.type=="InjectionReady").status', sbr_name, 'True')
     sbo_jq_is(context, '.status.conditions[] | select(.type=="Ready").status', sbr_name, 'True')
+    context.sb_secret = get_sbr_secret_name(context, sbr_name)
+
+
+@step(u'Service Binding secret is not present')
+def sb_secret_is_not_present(context):
+    openshift = Openshift()
+    output = openshift.search_resource_in_namespace("secrets", context.sb_secret, context.namespace.name)
+    assert output is None, f"Service Binding secret {context.sb_secret} is present in namespace '{context.namespace.name}'"
```

Comment:
this check needs to be performed with pooling - to allow cluster to remove the secret eventually.

Reply:
+1